libsplinter/src/auth/oauth/mod.rs

```diff
@@ -93,6 +93,43 @@ impl OAuthClient {
         })
     }
 
+    /// Creates a new `OAuthClient` with GitHub's authorization and token URLs.
+    ///
+    /// # Arguments
+    ///
+    /// * `client_id` - The GitHub OAuth client ID
+    /// * `client_secret` - The GitHub OAuth client secret
+    /// * `redirect_url` - The endpoint that GitHub will redirect to after it has completed
+    ///   authorization
+    #[cfg(feature = "oauth-github")]
+    pub fn github(
```

Comment:
Maybe call this new_with_github_defaults() or new_github().

Reply:
Renamed to `new_github`